components/ApplicationsList/ApplicationsList.jsx

```diff
@@ -37,6 +37,11 @@ const ApplicationsList = ({
         accessor: 'applicationDate',
         Cell: ({ value }) => new Date(value).toLocaleString(),
       },
+      {
+        Header: 'Exported for payment',
```

Comment:
Let's capitalise to maintain consistency across other column headers.

```suggestion
        Header: 'Exported for Payment',
```